charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.23.0
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 2.23.0
+version: 2.24.0
```

Comment:
Please remove this diff from your PR. We generally do not accept PRs that add a feature and perform a release at the same time - releases are performed by our maintainers on a semi-regular schedule and take into account details such as backwards compatibility, our support policy, updating the changelog, etc.

Other than that this PR lgtm.